longestname/script.js

```diff
@@ -0,0 +1,25 @@
+const name = "Jack" 
```

Comment:
Avoid using 'name' as a variable name in JavaScript. Some words are reserved for other uses in JS and should be avoided when naming variables. You can read about it here: 

https://www.w3schools.com/js/js_reserved.asp#:~:text=JavaScript%20Objects%2C%20Properties%2C%20and%20Methods

 'name1' would fit well with the other variable names.